src/vendor/Template.js

```diff
@@ -156,16 +169,10 @@ function expand(template, parameters) {
   return expandAny(template, [map]);
 }
 
-class Template {
-  constructor(template) {
-    this.template = template;
-  }
+Template.expand = expando;
 
-  expand(values) {
-    return expand(this.template, values);
-  }
-}
-
-Template.expand = expand;
+Template.prototype.expand = function expand(values) {
+  return expando(this.template, values);
+};
 
-export default Template;
+export { Template }; // eslint-disable-line import/prefer-default-export
```

Comment:
That export demands the import be changed to match; the named imports will no longer work.

I do not know why ES6 has two different types of imports, it it an unnecessary complication.  I do not like guessing if I should be importing a `default` export or a named export.

Since the `default` exports are less flexible, I try to stick with named exports for uniformity.